### PR TITLE
DM-38171: Move error_code_decisions.yaml to cm_prod and have reading it be part of campaign setup

### PR DESCRIPTION
**Pull request by eigerx**
This should mostly be done, but I'm requesting a review especially since I'm not sure why the last bit doesn't work. I did a small test of loading in the error code decision yaml at setup with hsc_rc2 and it just didn't do anything. Here is the workflow, which did make it to panda: https://panda-doma.cern.ch/tasks/?jeditaskid=151584|151586|151589|151588|151587|151585

### CODE REVIEW

**Comment by eigerx** [comment]
From @eacharles  in a slack dm: Ok,  you put the load-error-types after they daemon, but you are ctrl-c-ing out of the daemon, so the load-error-types never gets hit

I will move this one line earlier and test to see if that worked.

**Review by eacharles — APPROVED** [review]
Haha, now you get to learn how to deal with conflicts when doing a rebase.

File: examples/script_hsc_rc2_subset.sh [hunk 1]
```diff
@@ -4,12 +4,34 @@ EXAMPLES="$(dirname -- "$(readlink -f -- "$0";)";)"
 
 source $EXAMPLES/examples/hsc_rc2_subset_setup.sh
 
-\rm -rf $CM_PROD_URL/$fullname $db_path
-mkdir -p output
-
-cm create
-cm parse --config-name ${config_name} --config-yaml ${config}
-cm load-error-types --config-yaml examples/error_code_decisions.yaml
```
Comment (eigerx) [inline examples/script_hsc_rc2_subset.sh]: Oops I didn't mean to delete specifically this line! Let me fix this.

File: examples/script_hsc_weekly.sh [hunk 2]
```diff
@@ -4,12 +4,34 @@ EXAMPLES="$(dirname -- "$(readlink -f -- "$0";)";)"
 
 source $EXAMPLES/examples/hsc_weekly_setup.sh
 
-\rm -rf $CM_PROD_URL/$fullname $db_path
-mkdir -p output
-
-cm create
-cm parse --config-name ${config_name} --config-yaml ${config}
-cm load-error-types --config-yaml examples/error_code_decisions.yaml
```
Comment (eigerx) [inline examples/script_hsc_weekly.sh]: And this one!

**Pull request merged**